client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -205,7 +223,16 @@ const PatternAssembler: Step = ( { navigation } ) => {
 				<PatternSelectorLoader
 					showPatternSelectorType={ showPatternSelectorType }
 					onSelect={ onSelect }
-					onBack={ () => setShowPatternSelectorType( null ) }
+					onBack={ () => {
+						const uniquePatterns = uniqBy( getPatterns( showPatternSelectorType ), 'id' );
```

Comment:
It would be better to use [uniqueBy](https://github.com/Automattic/wp-calypso/blob/248e4c4557f20697ef825213472d95abc20112a7/packages/js-utils/src/unique-by.ts#L3) from `@automattic/js-utils` as we try to remove `lodash` from `wp-calypso` (But I'm not sure the current status 😅)